index.d.cts

```diff
@@ -0,0 +1,96 @@
+import {
+  CursorOptions,
+  CursorResult,
+  FileInfoOptions,
+  FileInfoResult,
+  Options,
+  ResolveConfigOptions,
+  doc,
+  util,
+} from "prettier";
+
+declare namespace prettierSync {
+  interface PrettierSync {
+    doc: typeof doc;
+
+    util: typeof util;
+
+    version: string;
+
+    /**
+     * `check` checks to see if the file has been formatted with Prettier given those options and returns a `Boolean`.
+     * This is similar to the `--list-different` parameter in the CLI and is useful for running Prettier in CI scenarios.
+     */
+    check: (source: string, options?: Options | undefined) => boolean;
```

Comment:
Will it work, if we use `check: Awaited<ReturnType<typeof check>>;`? This will be easier to maintain.

Reply:
Done. :)